src/content/docs/release-notes/agent-release-notes/ruby-release-notes/ruby-agent-9-15-0.mdx

```diff
@@ -0,0 +1,45 @@
+---
+subject: Ruby agent
+releaseDate: '2024-10-31'
+version: 9.15.0
+downloadLink: https://rubygems.org/downloads/newrelic_rpm-9.15.0.gem
+features: ["New configuration option cloud.aws.account_id", "Use default `View/component` metric name for unidentified View Components"]
+bugs: ["Instrumentation errors when using the karafka-rdkafka gem", "Stop calling deprecated all_specs method to check for the presence of newrelic-grape", "Instrumentation errors when using the ruby-kafka gem", "Restore AWS Lambda layer operational functionality"]
+security: []
+---
+
+<Callout variant="important">
+  We recommend updating to the latest agent version as soon as it's available. If you can't upgrade to the latest version, update your agents to a version no more than 90 days old. Read more about [keeping agents up to date](/docs/new-relic-solutions/new-relic-one/install-configure/update-new-relic-agent/).
+
+  See the New Relic Ruby agent [EOL policy](https://docs.newrelic.com/docs/apm/agents/ruby-agent/getting-started/ruby-agent-eol-policy/) for information about agent releases and support dates.
+</Callout>
+
+## v9.15.0
+
+Version 9.15.0 updates View Componment instrumentation to use a default metric name when one is unavailable, adds a configuration option to associate the AWS account ID with the DynamoDB calls from the AWS SDK, resolves a bug in rdkafka instrumentation when using the karafka-rdkafka gem, resolves a bug in the ruby-kafka instrumentation, fixes a bug with Grape instrumentation, and addresses a bug preventing the agent from running in serverless mode in an AWS Lambda layer.
+  
+- **Feature: New configuration option cloud.aws.account_id**
+
+  A new configuration option has been added, `cloud.aws.account_id`, that will allow New Relic to provide more details about certain calls made using the AWS SDK. One example, is that relationships between AWS services instrumented with New Relic's CloudWatch Metric Streams will have relationships formed in the service map with APM applications. Currently, the DynamoDB instrumentation is the only instrumentation that will make use of this configuration option, but this will be used in future instrumentation as well. [PR#2904](https://github.com/newrelic/newrelic-ruby-agent/pull/2904)
```

Comment:
```suggestion
  A new configuration option has been added, `cloud.aws.account_id`, that will allow New Relic to provide more details about certain calls made using the AWS SDK. For example, relationships between AWS services instrumented with New Relic's CloudWatch Metric Streams will have relationships formed in the service map with APM applications. Currently, the DynamoDB instrumentation is the only instrumentation that will make use of this configuration option, but this will be used in future instrumentation as well. [PR#2904](https://github.com/newrelic/newrelic-ruby-agent/pull/2904)
```